fundamentals-recap/math-primer.md

```diff
@@ -0,0 +1,175 @@
+# Math Primer
+
+## Goal
+
+Problem-solving in programming is almost always centered on _data_. To work with that data effectively, it can be useful to be aware of the following math concepts. By knowing the definitions of these concepts, they become problem-solving tools for us to use in the future.
+
+The goal of this document is to be a resource. We recommend spending your energy on this document reading through it and taking notes on it once, and then reviewing briefly every few months.
+
+This document covers the following:
+
+Thinking about numbers:
+
+1. Decimal Number System
+1. Using _n_ as a Variable in Math
+1. Coordinate Planes
+
+Working with collections of numbers:
+
+1. Maximum and Minimum
+1. Sorting
+1. Average or Mean
+1. Floor and Ceiling Functions
+
```

Comment:
Modulo would be a good addition in the future as it's often confused with division to newer students.